strings/strings.c

```diff
@@ -29,19 +29,19 @@ char *reverse_string(char *rv, char *s)
 {
     int sLen = string_length(s);
 
-    printf("rv start address: %c\n", &rv);
+    printf("rv start address: %p\n", rv);
 
-    for (int i = sLen; i >= 0; i--)
+    for (int i = sLen - 1; i >= 0; i--)
```

Comment:
I remember this, we solved it in standup :) this was a tough one!
remember to rely more on PythonTutor!

Reply:
I did finally get the VSCode debugger working this afternoon, but PythonTutor will make for a handy backup for smaller projects. :)